deliverables/README.md

```diff
@@ -31,38 +36,154 @@ What is the role of Javascript in a web page?
 
 #### meta
 Why do we separate HTML and CSS?  
+They are separated for organization (meaning) and clarity (presentation). 
 
 #### HTML
 What are classes and IDs (and how are they different)?  
+In the CSS, a class selector is a name preceded by a full stop (“.”) and an ID selector is a name preceded by a hash character (“#”).
+Classes are used for multiple items while IDs are used only once.
+
 What are elements?  
+Elements are tags and the content within them.
+
 What are tags?  
+The basic structure of an HTML document includes tags, which surround content and apply meaning to it.
+
 What are attributes?  
+Tags can also have attributes, which are extra bits of information. Attributes appear inside the opening tag and their values sit inside quotation marks. Their format is <tag attribute equals value
+
 What are forms?  
+They notate to take information and send it elsewhere. (i.e., collecting emails and sending it to a mailing list)
+
 What is a div?  
+Div is used to chunk code so information can be attached to it. 
+The difference between span and div is that a span element is in-line and usually used for a small chunk of HTML inside a line (such as inside a paragraph) whereas a div (division) element is block-line (which is basically equivalent to having a line-break before and after it) and used to group larger chunks of code.
 
 What are selectors?  
+Selectors are the names given to styles in internal and external style sheets. Whereas HTML has tags, CSS has selectors. 
+
 What are properties?  
-What are values?  
+For each selector there are “properties” inside curly brackets, which simply take the form of words such as color, font-weight or background-color.
+
+What are values? 
+A value is given to the property following a colon (NOT an “equals” sign) and semi-colons separate the properties.
+
 How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
+You use the inherit value in the author or user style sheet. The inheritance comes from the designated attribute created by the author.
```

Comment:
Also unless an element has a rule that matches it and applies a new value for a given property, the value of that property is inherited from its parent element.